app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt

```diff
@@ -198,7 +198,8 @@ class LoginPresenter(loginActivity: LoginActivity) : ILoginPresenter, ILoginMode
             if (CredentialHelper.isURLValid(url)) {
                 if (CredentialHelper.getValidURL(url) != null) {
```

Comment:
This is already checked above. Refactor into a val and you won't need anything else